documentation/release_notes.markdown

```diff
@@ -7,24 +7,311 @@ canonical: "/puppetdb/latest/release_notes.html"
 3.0.0
 -----
 
-### Upgrading
+### Before Upgrading
+
+* We have retired support for versions of PostgreSQL prior top 9.3. If you are using HSQLDB for production,
```

Comment:
I preferred the wording in the 2.3.0 release notes, it also covers the trgm extension installation in the same point.